modules/darwin/src/main/ts/ephox/darwin/mouse/MouseSelection.ts

```diff
@@ -25,8 +25,15 @@ export const MouseSelection = (bridge: WindowBridge, container: SugarElement<Nod
       findCell(event.target, isRoot).each((finish) => {
         CellSelection.identify(start, finish, isRoot).each((cellSel) => {
           const boxes = cellSel.boxes.getOr([]);
-          // Wait until we have more than one, otherwise you can't do text selection inside a cell.
-          if (boxes.length > 1) {
+          if (boxes.length === 1) {
+            // If a single CEF cell is selected, make sure it is annotated
+            const singleCell = boxes[0];
+            if (ContentEditable.getRaw(singleCell) === 'false') {
+              annotations.selectRange(container, boxes, singleCell, singleCell);
+              bridge.selectContents(singleCell);
```

Comment:
I mentioned this when talking about it Friday, but I don't like use doing this as it's an incorrect action since it's placing the selection inside of a CEF element. Additionally this needs a Darwin changelog entry.

- [ ] Ensure the cell is selected, not it's inner content
- [ ] Darwin changelog entry